### ISSUE TITLE
Review a demo board

### ISSUE THREAD

**Issue by BHydden** [online-go/online-go.com#1562]
**Is your feature request related to a problem? Please describe.**
IRL tournaments use demo boards to transpose pro games, and commentators want to review those demo boards so they can discuss different positions, but currently there is no way to review / clone a demo board

**Describe the solution you'd like**
Be able to review a demo board.

**Describe alternatives you've considered**
Be able to challenge yourself to a "game" (obviously unranked) 

**Additional context**

these recent forum threads 
re review-demo-board feature
 request-demo-board-clone-with-auto-update

 
From mod chat, also requested several times on EGF streams

December 6, 2021

[ 1:39]xhu98: At the moment it isn't possible to clone a demo board right?

[ 1:39]xhu98: Seems like this might be a useful feature to keep major tournament commentaries on OGS

[ 1:43]shinuito: Yeah it doesn’t seem possible to copy branches between tabs

[ 1:44]shinuito: Is the use case when one person is relaying the game and others want to run commentary on it?

[ 1:58]xhu98: shinuito: yep

[ 1:58]xhu98: This is usually the case for IRL tournaments

[ 1:58]xhu98: The European Grand Slam is happening in two weeks, so they asked me specifically for this

[ 1:58]xhu98: would be nice to keep that one on OGS

[ 3:14]BHydden: anoek xhu98's comments above also been requested several times by EGF

**Comment by BHydden** [comment]
Links to forum threads broke

https://forums.online-go.com/t/re-review-demo-board/39710?u=bhydden

https://forums.online-go.com/t/feature-request-demo-board-clone-with-auto-update/40100?u=bhydden

**Comment by JonKo314** [comment]
I don't think demo boards really fit the use case. Game recordings don't need variations. The remaining times of the players would be nice though (I'm imagining players using smart clocks connected to OGS in the future).

But it could be easier to "just add" reviews to demo boards (or clones which receive updates as the game progresses).

**Comment by BHydden** [comment]
I agree, I think the ability to create a self play game where one account plays as both black and white and all other game features exist normally would be best. It likely comes down to which feature path is easier to implement.

**Comment by vaurienLL** [comment]
Yup it clearly is an issue when the commentator is not the person who broadcast or if there are several commentators.
We just can't see real-time variations.

Looks like both "playing a self game" or "reviez any game/demo/review" would solve this kind of issue. =)
All the best.

**Comment by BHydden** [comment]
Interestingly the desire to[ be able to play a self game](https://forums.online-go.com/t/play-with-a-friend-on-a-phone/40324?u=bhydden) has come up on the forums for another reason, so if that path is possible it seems to fulfill multiple needs/desires/uses.

**Comment by cassidoo** [comment]
Bumping this to say that it would be great to make this possible! I'd love to be able to do this.

**Comment by vaurienLL** [comment]
I just up this a bit because we would love to have this feature so we can stream easily tournaments + live reviews
All the best :)

**Comment by shinuito** [comment]
I just want to reiterate the request for this for relaying tournament games to OGS. I think there’s a lot of convenience when reviewing a game being played on OGS, especially when streaming commentary because one can share a browser source for both the game and commentary without having to use a window or whole screen capture which can sometimes risk showing some personal information/things you don’t want to be shared if not careful.

With games being broadcast and scribed using demo boards, the current state of the game could be shared with a browser source, but the perspective and variations of a reviewer would still need a window or display capture, since they’re not in control of the current state of the demo boards.

**Comment by Skwidder** [comment]
I Will bump this aswell, I'm building a tool that would really benefit from this feature. and have started looking at KGS because of it being missing. 

I think this is critical for streaming and i worry if it doesn't get added soon OGS may lose a lot of broadcasters
